src/mixins/state.js

```diff
@@ -46,34 +58,57 @@ export default {
   },
 
   /**
-   * Unbind all entity events and remove any listeners on _stateModel
-   * Clean up destroy event handler
+   * Bind events from the _stateModel defined in stateEvents hash
+   * Setup destroy event handle
    *
-   * @private
-   * @method _removeEventHandlers
+   * @public
+   * @method delegateStateEvents
```

Comment:
Added docs for `delegateStateEvents`/`undelegateStateEvents` and double checked that we've got 100% coverage.